daprdocs/content/en/reference/components-reference/supported-state-stores/setup-sqlserver.md

```diff
@@ -52,8 +52,8 @@ If you wish to use SQL server as an [actor state store]({{< ref "state_api.md#co
 
 | Field              | Required | Details | Example |
 |--------------------|:--------:|---------|---------|
-| connectionString   | Y        | The connection string used to connect | `"Server=myServerName\myInstanceName;Database=myDataBase;User Id=myUsername;Password=myPassword;"`
-| tableName          | Y        | The name of the table to use. Alpha-numeric with underscores | `"table_name"`
+| connectionString   | Y        | The connection string used to connect. If the connection string contains the database it must already exist. If the database is omitted a default database named `"Dapr"` will be created.  | `"Server=myServerName\myInstanceName;Database=myDataBase;User Id=myUsername;Password=myPassword;"`
```

Comment:
:a default database named `"Dapr"` will be created - I tend to prefer the present tense in these as in "a default database named `"Dapr"` is created.